README.md

```diff
@@ -17,6 +17,7 @@ Feel free to add more languages. Please follow the requirements in each subdirec
 | ---- | ----------- | ----- |
 | Outbound Dial | Uses the [StartOutboundVoiceContact](https://docs.aws.amazon.com/connect/latest/APIReference/API_StartOutboundVoiceContact.html) API to call a number. | [Java](java/OutboundExample) [DotNet](dotnet/OutboundExample) |
 | Holiday Check | Can be used to check for holidays in a contact flow | [Python](python/holidaycheck) |
+| Sync Instances | Can be used to sync basic user information from one instance to another instance | [Python](python/syncinstances) |
```

Comment:
This title is misleading. I expected it to sync the entire instance. "Sync Users in Instance" or such would be better